src/AWS.Deploy.Common/Recipes/PropertyDependency.cs

```diff
@@ -9,14 +9,22 @@ namespace AWS.Deploy.Common.Recipes
     public class PropertyDependency
     {
         public string Id { get; set; }
+        public PropertyDependencyOperationType? Operation { get; set; }
         public object Value { get; set; }
 
         public PropertyDependency(
             string id,
+            PropertyDependencyOperationType? operation,
             object value)
         {
             Id = id;
+            Operation = operation;
             Value = value;
         }
     }
+
+    public enum PropertyDependencyOperationType
```

Comment:
Is the Toolkit (a server mode consumer) going to need to understand how to work with this?

Reply:
The setting dependency is abstracted and the toolkit will not need to interface with these properties.

Reply:
Could you improve documentation here anyway? As a new developer working on the tool this isn't clear to me, and I imagine developers writing custom recipes may need to understand this.
* Document enum values here
* Add a property for this to aws-deploy-recipe.schema-json (with a description, and backfill descriptions for `Id` and `value` too